src/link/script.y

```diff
@@ -411,6 +469,16 @@ static void alignTo(uint32_t alignment, uint32_t alignOfs) {
 
 static void pad(uint32_t length) {
 	auto const &context = lexerStack.back();
+	if (activeType == SECTTYPE_INVALID) {
+		scriptError(context, "Cannot pad: no memory region is active");
```

Comment:
"Pad" is not the verb used in the documentation for `ds`, so you'll want to make these match up.